### ISSUE TITLE
Plugin fails if logged in to FB through iOS Settings

### ISSUE THREAD

**Issue by dregre** [Wizcorp/phonegap-facebook-plugin#604]
Hello--

I have tried finding a solution to this problem, to no avail, and am wondering whether others have been having the same problem.

When a user logs into Facebook under iOS' settings (i.e. not through the FB app), and then tries to login through Cordova application with the Facebook plugin, nothing happens. No error is printed to the Objective-C console nor to the JS console.  However, when the user is not logged into Facebook through iOS' settings, the plugin works as expected: it will redirect him to the FB app, if installed, or open a webview.

So, in summary: When a user logs in through the FB app (or a web view), the plugin works. When a user logs in through iOS settings, the plugin does not work.

Thoughts?

Best,
amgreg
